### DISCUSSION TITLE
Unique authentication credentials for each tenant?

### DISCUSSION THREAD

**Discussion by justaskz** [grafana/mimir#9678] [General]
Hello all, I have a question - is there a way to have unique authentication credentials for each tenant? There is no straightforward way, but maybe I am missing some kind of workaround?

**Comment by narqo** [comment]
For open-source Mimir, have a look at [Grafana Mimir authentication and authorization][1]. For GEM (Grafana Enterprise Metrics) the [Managing tenants in GEM][2] provides more options built-in.

If the above pieces of documentation aren't sufficient, please explain your use-cases with mode details.

[1]: https://grafana.com/docs/mimir/latest/manage/secure/authentication-and-authorization/#grafana-mimir-authentication-and-authorization
[2]: https://grafana.com/docs/enterprise-metrics/latest/tenant-management/
